e2e/ls_test.go

```diff
@@ -139,6 +141,30 @@ func TestListSingleWildcardS3Object(t *testing.T) {
 	}, alignment(true))
 }
 
+// ls -s bucket/object
+func TestListSingleWildcardS3Object2(t *testing.T) {
+	t.Parallel()
+
+	bucket := s3BucketFromTestName(t)
+
+	s3client, s5cmd, cleanup := setup(t)
+	defer cleanup()
+
+	createBucket(t, s3client, bucket)
+	putFile(t, s3client, bucket, "testfile1.txt", "this is a file content")
+
+	cmd := s5cmd("ls -s", "s3://"+bucket+"/testfile1.txt")
+	result := icmd.RunCmd(cmd)
+
+	result.Assert(t, icmd.Success)
+
+	// on mock s3 client, storage class short code is represented by `?`
+	// storage class full display is an empty string.
+	stclass := !strings.Contains(result.Stdout(), "?")
```

Comment:
I think we should not assert the output of the storage class since it is not correctly set by gofakes3. If we add this test and assert `?` or `empty strings` for stdout, we gonna test the behavior of gofakes3 rather than s5cmd. This test can be added in the future if they support storage classes. We can just test exitcode of command and skip other assertions:

```
	cmd := s5cmd("ls", "-s", "s3://"+bucket+"/testfile1.txt")
	result := icmd.RunCmd(cmd)

	result.Assert(t, icmd.Success)
```